include/mbedtls/debug.h

```diff
@@ -108,16 +108,16 @@
  *
  * This module provides debugging functions.
  */
-#if (defined(__MINGW32__) && __USE_MINGW_ANSI_STDIO == 0) || (defined(_MSC_VER) && _MSC_VER < 1800)
+#if defined(__MINGW32__) || (defined(_MSC_VER) && _MSC_VER < 1900)
```

Comment:
https://learn.microsoft.com/en-us/cpp/c-runtime-library/format-specification-syntax-printf-and-wprintf-functions?view=msvc-140#size-prefixes-for-printf-and-wprintf-format-type-specifiers and https://stackoverflow.com/questions/15610053/correct-printf-format-specifier-for-size-t-zu-or-iu/44055215#44055215 confirm that `_MSC_VER == 1900` is enough to get `%zu` and `%lld`.